src/deploy/functions/runtimes/python/index.ts

```diff
@@ -143,6 +143,8 @@ export class Delegate implements runtimes.RuntimeDelegate {
     const envWithAdminPort = {
       ...envs,
       ADMIN_PORT: port.toString(),
+      // Allow imports to work normally
+      PYTHONPATH: this.sourceDir,
```

Comment:
@taeold is there a reason why $PYTHONPATH was excluded during discovery and the emulators? it was making it so imports in my main.py were erroring

Reply:
I expected `venv` to take care of this, so I'm surprised that this works. I going to have to learn about PYTHONPATH and why this is solving the problem you are seeing. 🤔

Reply:
Yeah, ultimately we need to set the root path, if it's not cwd to allow imports from other folders. It is surprising that venv doesn't handle, maybe `add2virtualenv` is the correct way to handle. My python is very rusty though.